src/PerfView/PerfView.csproj

```diff
@@ -510,6 +511,13 @@
       <Link>Microsoft.Identity.Client.Extensions.Msal.dll</Link>
       <Visible>False</Visible>
     </EmbeddedResource>
+    <EmbeddedResource Include="$(PkgMicrosoft_IdentityModel_Abstractions)\lib\net461\Microsoft.IdentityModel.Abstractions.dll">
```

Comment:
@brianrob Not on this PR specifically, but the embedding of dependencies can be automated so that you don't have to do the "GeneratePathProperty" thing and we never have missing embedded dependencies. My (increasingly fallible) memory tells me I've showed you how to do this before and you weren't interested. Are you interested now?

Reply:
If I recall correctly, the issue I ran into with this last time was that the set of binaries was much larger than we actually need for PerfView (though probably not if we're being exhaustive with regard to TraceEvent).  Can you remind me how this works?

Reply:
It was some extra MSBuild logic that we used in the Snapshot Collector for the same purpose.
Yes, it does result in a larger set. It's actually the complete, transitive set of dependencies -- according to the build. As you say, that's probably larger than strictly needed at runtime. I reason it is better to carry the complete set than to hope you can discover missing references by manual testing.

Reply:
I looked for the commit that had it in, but I can't find it. I must've deleted the old branch where I did the work. I can try and recreate it in a different PR.